website/docs/artifacts.md

```diff
@@ -332,3 +332,28 @@ This is a global setting that applies to all artifacts only.
 
 If you want some binaries stripped and others not, you will need to manually
 strip them in the build phase.
+
+## Automatic Dependency Resolution
+
+Some package tooling, such as `rpmbuild` or `debbuild` both used in core DALEC,
+will attempt to automatically resolve runtime dependencies for you based on
+the artifacts you have specified.
+This can be linked libraries or even detecting you've included a shell script
```

Comment:
The wording for this line a bit confusing
```suggestion
This can be linked libraries or even detecting you've included a shell script
```

Reply:
I'm not seeing a diff in that suggestion.

Reply:
yeah sorry, I don't have a suggested fix but I did want to point out that is a bit confusing

Reply:
maybe if I understand correctly
```suggestion
This can be from linking libraries or even detecting you've included a shell script
```